client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -349,6 +350,14 @@ const siteSetupFlow: FlowV1 = {
 				}
 
 				case 'design-choices': {
+					if ( providedDependencies.destination === 'launch-big-sky' ) {
+						const queryParams = new URLSearchParams( location.search ).toString();
+						calypsoLibNavigate(
+							`/setup/site-setup/launch-big-sky${ queryParams ? `?${ queryParams }` : '' }`
+						);
+						return;
+					}
```

Comment:
This is the logic that used to live directly inside the design choices step component.